src/components/SRNEntry.vue

```diff
@@ -164,64 +179,87 @@ export default {
         },
       ];
     },
+    /* checks to see if the input character is a number. Makes use of ASCII values.
+    @param {Event} - e - event triggered when a character is typed.  */
```

Comment:
```suggestion
    @param {Object} - e - event triggered when a character is typed.  */
```
There's no data type called as `Event` right?
It's basically an Object that's being passed into `e`